cocos/editor-support/cocostudio/CCSSceneReader.cpp

```diff
@@ -364,6 +364,10 @@ cocos2d::Node* SceneReader::createObject(CocoLoader *cocoLoader, stExpCocoNode *
     {
         className = pNodeArray[1].GetValue(cocoLoader);
     }
+    if (className == nullptr) {
```

Comment:
Maybe it's true here. But AFAIK nobody report this issue. I think modern compiler will take care of this issue.